### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by President10000** [community/community#58585] [New to GitHub]
?

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @President10000  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
